molecule/verifier/testinfra.py

```diff
@@ -55,7 +55,8 @@ def execute(self):
             self._molecule.verifier_options)
         testinfra_options['env'] = ansible.env
         testinfra_options['debug'] = self._molecule.args.get('debug', False)
-        testinfra_options['sudo'] = self._molecule.args.get('sudo', False)
+        if self._molecule.args.get('sudo'):
+            testinfra_options['sudo'] = True
```

Comment:
So this one is really just not a True/False? Meaning that 'sudo' will never be present in self._molecule.args with a value of False?